city_scrapers/spiders/chi_ssa_18.py

```diff
@@ -0,0 +1,117 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa18Spider(CityScrapersSpider):
+    name = "chi_ssa_18"
+    agency = "Chicago Special Service Area #18 North Halsted"
+    timezone = "America/Chicago"
+    start_urls = ["https://northalsted.com/community/"]
+    location = {
+        'name': 'Center on Halsted',
+        'address': '3656 N Halsted St, Chicago IL 60613, Conference Room 200 on the 2nd Floor'
```

Comment:
To keep things consistent this can be "3656 N Halsted St, Conference Room 200, Chicago IL 60613"

Reply:
updated